src/Icicle/Avalanche/Statement/Simp/Dead.hs

```diff
@@ -40,70 +44,71 @@ instance Eq n => Monoid (Usage n) where
   , usageExp = Set.union (usageExp a) (usageExp b)
   }
```

Comment:
no - you are right actually. I just saw that it doesn't thread through quite the same way after I commented